Yoast/javascript
# Improve the stemming of all the forms of the entries in exception lists

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Improve the stemming of all the forms of the entries in the exception lists

## Relevant technical choices
* This PR is a shadow of https://github.com/Yoast/javascript/pull/542, where things went wrong with merging and merge conflicts resolution.

## Test instructions
This PR can be tested by following these steps:

* Add other forms of the exception lists entries to the determineSpec, for example from removeSuffixFromFullForms list. e.g. benefietje, benefietjes
* Run `yarn test` and make sure there is no failing test. 


## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-305
